app/views/components/blockRegistration.html

```diff
@@ -1,8 +1,8 @@
 <div class="form-group">
-  <label class="control-label">{{block.title}} <em class="required" ng-if="block.required">*</em></label>
+  <label class="control-label" ng-bind-html="block.title"><em class="required" ng-if="block.required">*</em></label>
```

Comment:
Same thing here:
`ng-bind-html` seems to be overwriting the `em` that indicates when a block is required. Maybe add an inner `<span>` and bind to that?